### PR TITLE
Create 2015-06-18-bubble-color

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp

### CODE REVIEW

**Comment by chriddyp** [comment]
:dancer: , and let's remove this one: https://github.com/plotly/documentation/blob/gh-pages/_posts/python/bubble/2015-04-09-bubblechart.html now that we have these. 
we should also probably include a `color` example that uses numerical values and the colorscale instead of just color strings

File: _posts/python/bubble/2015-06-18-bubble-color [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Marker Size & Color Styling for Bubble Chart
```
Comment (chriddyp) [inline _posts/python/bubble/2015-06-18-bubble-color]: maybe just `Setting Marker Size and Color` since we're already in the bubble chart section?

**Comment by aneda** [comment]
@chriddyp  do you mean something like this https://plot.ly/3730/~etpinard/ 
this has both colorscale and color as number
ps. I merged the bubble-text here and will correct it later

**Comment by chriddyp** [comment]
@aneda yeah, exactly

**Comment by aneda** [comment]
@chriddyp ok sounds good.   I think we should delete this one as well:
https://github.com/plotly/documentation/blob/gh-pages/_posts/python/bubble/2015-04-09-bubble-chart-sizeref-sizemode.html

**Comment by chriddyp** [comment]
@aneda that sounds good, I'd still love to see a simple illustrative example of how `sizeref` actually works

**Comment by aneda** [comment]
@chriddyp you mean another simple example for sizeref beside this one:

image

https://github.com/plotly/documentation/blob/gh-pages/_posts/python/bubble/2015-06-18-bubble-marker-size

**Comment by chriddyp** [comment]
ah nvm, I missed that sizeref one. cool, good to remove https://github.com/plotly/documentation/blob/gh-pages/_posts/python/bubble/2015-04-09-bubble-chart-sizeref-sizemode.html? and merge?

**Pull request merged**